ci/access-control-allow-all.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -ex
+
+ipfs config --json API.HTTPHeaders.Access-Control-Allow-Origin '["*"]'
+ipfs config --json API.HTTPHeaders.Access-Control-Allow-Methods '["*"]'
```

Comment:
Anything in particular we're concerned with? The reason why this is needed is because in the "everything in docker" setup, the browsers run in different containers than kubo. There might be another way to do it. This was just the one that required the least thought.